test/real_cases_periodic_test.rb

```diff
@@ -163,7 +163,7 @@ def test_performance_13vl
 
       # voluntarily equal to watch evolution of periodic algorithm performance
       assert_equal expected, seen, 'Do not have the expected number of total visits'
-      assert_equal 301, unassigned_visits.sum, 'Do not have the expected number of unassigned visits'
```

Comment:
any idea where the improvement comes from ?

Reply:
My assumption is that before services at the same position but non related were performed in any case in the same row even if there was no particular reason to do so. Now there is no discrimination between these and their neighbors. So we can explore solutions with a more "global" perspective.

Reply:
This is great news!